src/server/models/friendsSchema.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+const friendsSchema = new Schema({
```

Comment:
Isn't it a `friend`? I don's see any list here.

Reply:
ok